### PR TITLE
Fixed the refresh button and improved the loading animation

### PR DESCRIPTION
**Pull request by jsanchez91**
Hi team!

Fixed the update button in the Miter Frame component and the compliance inventory component.
On the other hand, we made changes in the loading animation in both components.
refresh_button

Regards.

### CODE REVIEW

File: public/components/overview/mitre/components/tactics/tactics.tsx [hunk 1]
```diff
@@ -138,14 +139,15 @@ export class Tactics extends Component {
...
+      const { data } = await getElasticAlerts(indexPattern, filterParams, aggs);
       const { buckets } = data.aggregations.tactics;
       if(firstTime){
-       this.initTactics(buckets); // top tactics are checked on component mount
+        this.initTactics(buckets); // top tactics are checked on component mount
       }
       this._isMount && this.setState({tacticsCount: buckets, loadingAlerts: false, firstTime:false});
         
     } catch(err){
+      console.log(err)
```
Comment (juankaromo) [inline public/components/overview/mitre/components/tactics/tactics.tsx:150]: Is it necessary?

File: public/components/overview/mitre/components/techniques/techniques.tsx [hunk 2]
```diff
@@ -79,12 +78,25 @@ export const Techniques = withWindowSize(class Techniques extends Component {
   
   async componentDidMount(){
     this._isMount = true;
-    await this.getTechniquesCount();
+    // await this.getTechniquesCount();
```
Comment (juankaromo) [inline public/components/overview/mitre/components/techniques/techniques.tsx:81]: Let's not leave things with comments

Comment (jsanchez91) [reply public/components/overview/mitre/components/techniques/techniques.tsx:81]: I delete it

File: public/components/overview/mitre/components/tactics/tactics.tsx (hunk 1 above)
Comment (jsanchez91) [reply public/components/overview/mitre/components/tactics/tactics.tsx]: Removed

**Pull request merged**